packages/design-system-tokens/src/themes/healthcare-components.ts

```diff
@@ -0,0 +1,254 @@
+/*
+ * Healthcare.gov CMSDS Component Theme
+ */
+
+import { default as t } from './healthcare'
+import { hexOpacity } from '../lib/utility'
+
+export const components = {
+  'alert': {
+    '__background-color':                         t.color['primary-alt-lightest'],
+    '__background-color--error':                  t.color['error-lightest'],
+    '__background-color--lightweight':            t.color['white'],
```

Comment:
This formatting and grouping is much easier to read, and I also like the succinct `t.*` shorthand.